src/base/__tests__/Context.ts

```diff
@@ -56,7 +56,6 @@ jest.mock(
   require('~/testUtils/mocks/entities').mockSubsidyModule('~/api/entities/Subsidy')
 );
 
-// TODO: refactor tests (too much repeated code)
```

Comment:
If anything this should be an issue, but I think they are OK even if a bit verbose.